src/Core/Checkout/Cart/LineItemFactoryRegistry.php

```diff
@@ -83,7 +83,7 @@ public function update(Cart $cart, array $data, SalesChannelContext $context): v
         if (isset($data['quantity'])) {
             $lineItem->setQuantity($data['quantity']);
 
-            $this->eventDispatcher->dispatch(new LineItemQuantityChangedEvent($lineItem, $cart, $context));
+            $this->eventDispatcher->dispatch(new BeforeLineItemQuantityChangedEvent($lineItem, $cart, $context));
```

Comment:
the old events still need to be dispatched. otherwise this would be breaking for plugins subscribing to those events